<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QWidget" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="controlGroup">
     <property name="title">
      <string>Control Panel</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="conveyerCountLayout">
        <item>
         <widget class="QLabel" name="conveyerLab">
          <property name="text">
           <string>Current Conveyer Count: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="conveyerLine"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="increasePB">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="decreasePB">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
